library/psa_crypto.c

```diff
@@ -6477,30 +6470,49 @@ static psa_status_t psa_pbkdf2_set_salt(psa_pbkdf2_key_derivation_t *pbkdf2,
     return PSA_SUCCESS;
 }
 
+static psa_status_t psa_pbkdf2_hmac_set_password(psa_algorithm_t hash_alg,
+                                                 const uint8_t *input,
+                                                 size_t input_len,
+                                                 uint8_t *output,
+                                                 size_t output_len)
```

Comment:
It seems that `output_len` is intended as an output parameter, so:
```suggestion
                                                 size_t *output_len)
```
and further changes in this function and its caller.